packages/@react-spectrum/tabs/stories/Tabs.stories.js

```diff
@@ -22,10 +22,10 @@ import {Tabs} from '..';
 storiesOf('Tabs', module)
   .add(
     'Default',
-    () => render(),
-  {
-    info: 'The page, anchored, and panel variants have been deprecated. Page is now compact, and panel/anchored tabs are just default. These variants will map properly to the new styles, but please do not specify these variants going forward.'
-  })
+    () => render(), {
+      info: 'The page, anchored, and panel variants have been deprecated. Page is now compact, and panel/anchored tabs are just default. These variants will map properly to the new styles, but please do not specify these variants going forward.'
```

Comment:
note?

Reply:
I went with `info` because the addon in v2 used that key. It also fell more in line with logging error, warning, and info. But this feature is for `Notes` which is why your suggestion makes sense. Anyone else have a preference on notes vs. info?

Reply:
I think putting it as a note is better, but do we need this note? Is it still relevant? Kinda reads like a v2 note (geared towards end users looking to the storybook as a source of truth, but in v3 land that becomes a less common occurrence)

Reply:
I misspoke, we do use note, and I'll remove this.